### ISSUE TITLE
tfdml_plugin.dll not found

### ISSUE THREAD

**Issue by halfman2016** [microsoft/tensorflow-directml-plugin#331]
tensorflow.python.framework.errors_impl.NotFoundError: C:\ProgramData\Miniconda3\envs\tfdml_plugin\lib\site-packages\tensorflow-plugins\tfdml_plugin.dll not found

when i install,run any code, like import tensorflow as tf,it is error ,not found tfdml_plugin.dll .  but,the file is right in the path.
how can i do ?

**Comment by halfman2016** [comment]
N卡，A卡都一样，系统win11

**Comment by phgilde** [comment]
I have this issue too. 
Windows 10, Version 22H2 Build 19045.2251
Radeon RX570 GPU with newest drivers.

**Comment by phgilde** [comment]
I could fix the issue by installing tensorflow-cpu==2.10 with the --user flag.

**Comment by Zhaopudark** [comment]
The same bug as mine, I'll try to build TensorFlow-directml-plugin tomorrow, anyway today is too late.

**Comment by PatriceVignola** [comment]
Yes, we recently found that TF 2.11 is not backward compatible with plugins built for earlier versions. Since pluggable devices are still experimental, this is to be expected. In the future, we'll hard lock the version of TF until pluggable devices are in a more stable state.

We will soon be releasing a new version of tensorflow-directml-plugin that addresses the problem. In the meantime, please manually install tensorflow-cpu==2.10 to get rid of the problem.

**Comment by KiTant** [comment]
I still have this error. I installed tensorflow-cpu==2.10.0 with the flag --user and without this flag but I still have this error: tfdml_plugin.dll not found.

**Comment by chanangaza** [comment]
Try uninstalling all of Keras and Tensorflow libs (including Tensorboard). Then reinstall tensorflow-directml-plugin.

Worked for me.

**Comment by KiTant** [comment]
That worked thx

But i getting other error when training AI:
```python
No OpKernel was registered to support Op 'CudnnRNN' used by {{node CudnnRNN}} with these attrs: [seed=0, dropout=0, T=DT_FLOAT, input_mode="linear_input", direction="unidirectional", rnn_mode="gru", seed2=0, is_training=true]
Registered devices: [CPU, GPU]
Registered kernels:
  <no registered kernels>

	 [[CudnnRNN]]
	 [[model/gru/PartitionedCall]] [Op:__inference_train_function_5775]
```

**Comment by TeaTimeChimp** [comment]
Re. CudnnRNN, sounds similar to this issue I commented on https://github.com/microsoft/tensorflow-directml-plugin/issues/364
